Tests/Shared/extension/QueueMockExtensions.swift

```diff
@@ -8,4 +8,13 @@ public extension QueueMock {
             ($0.data.value as? DeletePushNotificationQueueTaskData)?.deviceToken
         }.mapNonNil()
     }
+
+    var addTaskReturnValue: ModifyQueueResult {
+        get { fatalError("this is a setter only property") }
+        set {
+            addTaskClosure = { _, _, _, _, onComplete in
+                onComplete(newValue)
```

Comment:
where does `newValue` come from?

Reply:
It's a Swift lang feature. `newValue` [is a shorthand](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/properties/#Shorthand-Setter-Declaration) for the setter. It's the new value of the property when set.